src/expandableCalendar/calendarProvider.js

```diff
@@ -119,7 +122,7 @@ class CalendarProvider extends Component {
       const isToday = today === date;
       
       Animated.spring(this.state.buttonY, {
-        toValue: isToday ? 65 : -65,
+        toValue: isToday ? TOP_POSITION : -this.props.buttonTopPosition || -TOP_POSITION,
```

Comment:
What happens when `isToday` is false that you need a different value?

Reply:
I need it to get out of the frame. It won't with every value so I'm keeping my own value